src/System.Net.Security/src/System/Net/Security/InternalNegotiateStream.cs

```diff
@@ -264,7 +263,7 @@ private int StartFrameHeader(byte[] buffer, int offset, int count, AsyncProtocol
             if (asyncRequest != null)
             {
                 asyncRequest.SetNextRequest(_ReadHeader, 0, _ReadHeader.Length, s_readCallback);
-                FixedSizeReader.ReadPacketAsync(InnerStream, asyncRequest);
+                Task ignored = FixedSizeReader.ReadPacketAsync(InnerStream, asyncRequest);
```

Comment:
nit: Use discards? `_ = FixedSizeReader.ReadPacketAsync(...)`

Reply:
I hadn't considered that.  Changed. Thanks.